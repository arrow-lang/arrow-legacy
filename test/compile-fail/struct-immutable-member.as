struct Box { value: int }
let a = Box(10);
a.value = 340; // error
