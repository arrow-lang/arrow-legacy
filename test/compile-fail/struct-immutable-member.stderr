[0;37mtest/compile-fail/struct-immutable-member.as:3.1-14: [0;31merror: [1;37mcannot assign to immutable member 'value'[0m
